docs/blog.md

```diff
@@ -343,3 +343,10 @@ username 为博主用户名，而非`xxx.hashnode.dev`中`xxx`所代表的 blog
 :::
 
 </Route>
+
+
+## CSDN
+
+### 用户博客
+
+<Route author="Jkker" example="/csdn/blog/csdngeeknews" path="/csdn/blog/:user" radar="1" />
```

Comment:
Missing `paramsDesc` attribute

Reply:
Thanks for pointing this out. The `paramsDesc` attribute has been added in [the latest commit](https://github.com/DIYgod/RSSHub/pull/10447/commits/4d3107778a4cee2625c7fc124c1c6b6033156da9)